assets/src/edit-story/components/inspector/document/test/publish.js

```diff
@@ -92,12 +106,14 @@ describe('PublishPanel', () => {
     expect(element).toBeNull();
   });
 
-  it('should open Date picker when clicking on date', () => {
-    const { getByText, getByRole } = setupPanel();
+  it('should open sidebar when clicking on the date field', () => {
+    const { getByText, showSidebarAt } = setupPanel();
     const element = getByText('01/01/2020');
 
     fireEvent.click(element);
-    const calendar = getByRole('application');
-    expect(calendar).toBeDefined();
+    expect(showSidebarAt).toHaveBeenCalledWith(
+      expect.any(Object),
+      expect.any(Function)
```

Comment:
Also here I'd like to see at least some validation, that the right thing is displayed. This will have many false positives.